Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -23,12 +25,23 @@
     }
 </style>
 <div class="text-center">
-    <h1 class="display-4">Pfizer Vaccine Allocations For All States: <h3>@currentWeek</h3></h1>
+    <h1 class="display-4">Pfizer Vaccine Allocations: </h1>
+    
+    <br />
+    @if (@jurisdiction is null)
+    {
+        <h3> Vaccine distribution on @currentWeek.ToString().Substring(0, 9) by states</h3>
```

Comment:
Week of Allocation heading was taking the time value (time of allocation) which was not required. So, I formatted it to show just the day, month and year of allocation.

Source: https://docs.microsoft.com/en-us/dotnet/api/system.datetime.toshortdatestring?view=net-5.0